src/confluent_kafka/src/Admin.c

```diff
@@ -3892,6 +3893,15 @@ static PyObject *Admin_c_MemberDescription_to_py(const rd_kafka_MemberDescriptio
 
         PyDict_SetItemString(kwargs, "assignment", assignment);
 
+        c_target_assignment = rd_kafka_MemberDescription_target_assignment(c_member);
+        if(c_target_assignment) {
+                PyObject *target_assignment = Admin_c_MemberAssignment_to_py(c_target_assignment);
+                if (!target_assignment) {
```

Comment:
Check NULL case is working or not. target_assignment should be None in that case.